commands/stonks.js

```diff
@@ -18,14 +18,29 @@ exports.run = async (client, message, args, level) => {
       if(Object.keys(data).length === 0) {
         message.reply(`Invalid ticker [${company}]`);
       } else {
-        message.channel.send(`\`\`\`\n
+        const priceDiffPercent = Math.round((data.c - data.pc) / data.pc * 10000)/100;
+        const priceDiff = Math.abs(Math.round((data.c - data.pc)*100)/100);
+
+        let arrowDirection = '\u25B2'; //up
+        let pricePrefix = '\+';
+        if(priceDiffPercent < 0) {
+          arrowDirection = '\u25BC'; //down 
+          pricePrefix = '\-';
+        } 
+
+        // important to have price prefix match with diff markdown colors
+        // + in the beginning of the line for green
+        // - for red
+        message.channel.send(`\`\`\`diff
```

Comment:
Turns out you can't do normal text coloring but its only using markdown "hacking". 

Here I'm leveraging diff styling to get +green and -red colors. It only highlights when the line starts with +/-

Reply:
Sample output:
![image](https://user-images.githubusercontent.com/1926395/87634038-5e50e800-c6f1-11ea-8ab4-ebc0b9becb29.png)

Reply:
the green is not as green as the red is red, you get what I mean?

Reply:
Ya but I can't even choose the colors :(

Reply:
where are the colors defined?

Reply:
https://gist.github.com/matthewzring/9f7bbfd102003963f9be7dbcf7d40e51#syntax-highlighting

Reply:
I get what you mean

Reply:
They don't think it be like it is but it do.